### PR TITLE
Install dependencies of TF 1.11

### PR DESCRIPTION
**Pull request by petrbel**

### CODE REVIEW

File: dockerfiles/emloop-tensorflow [hunk 1]
```diff
@@ -11,6 +11,8 @@ RUN if [ "${tag}" = "cuda" ]; then \
         pacman --noconfirm --needed -S python-tensorflow-opt; \
     fi \
     && pacman --noconfirm --needed -S tensorboard \
+    && sudo -u aur gpg --recv-keys 8C004C2F93481F6B \
+    && sudo -u aur trizen --noconfirm --needed -S python-gast python-astor python-grpcio python-termcolor tensorboard \
```
Comment (FloopCZ) [inline dockerfiles/emloop-tensorflow]: `tensorboard` is already installed on L13

Comment (petrbel) [reply dockerfiles/emloop-tensorflow]: ammended, good catch

**Review by FloopCZ — APPROVED** [review]
Let's hope it will get into the official repo as dependencies. By the way, how do you know that those are the dependencies? I can import `tensorflow` even without those.

**Comment by petrbel** [comment]
Installation of tensorflow yielded warnings that these are missing. Without them, I wasn't able to use (already trained) network for prediction. Installation fixed it

**Pull request merged**